content/Video_Display.md

```diff
@@ -127,9 +127,9 @@ drawing exceeds the lower (right) border of the BG map area.
 
 ### FF44 - LY (LCDC Y-Coordinate) (R)
 
-The LY indicates the vertical line to which the present data is
-transferred to the LCD Driver. The LY can take on any value between 0
-through 153. The values between 144 and 153 indicate the V-Blank period.
+LY indicates the current horizontal line, which might be about to be drawn,
+being drawn, or just been drawn. LY can take
+on any value from 0 to 153. The values from 144 to 153 indicate the V-Blank period.
```

Comment:
I'm not sure that "to take *on* a value" is appropriate here? (Genuinely asking, I'm no native speaker.)

Reply:
Yeah I didn't like it either. I have replaced it with "hold".